ngraph/core/src/op/convert.cpp

```diff
@@ -180,6 +180,52 @@ bool op::v0::Convert::evaluate(const HostTensorVector& output_values,
     return convert::evaluate_convert(input_values[0], output_values[0]);
 }
 
+bool op::v0::Convert::has_evaluate() const
+{
+    NGRAPH_OP_SCOPE(v0_Convert_has_evaluate);
+
+    bool hasEvaluate = false;
+    switch (get_input_element_type(0))
+    {
+    case ngraph::element::u1:
+    case ngraph::element::u4:
+    case ngraph::element::u8:
+    case ngraph::element::u16:
+    case ngraph::element::u32:
+    case ngraph::element::u64:
+    case ngraph::element::i4:
+    case ngraph::element::i8:
+    case ngraph::element::i16:
+    case ngraph::element::i32:
+    case ngraph::element::i64:
+    case ngraph::element::bf16:
+    case ngraph::element::f16:
+    case ngraph::element::f32:
+    case ngraph::element::boolean: hasEvaluate = true; break;
```

Comment:
nitpick: we don't need to have `hasEvaluate` variable, because you always return `false` for unsupported cases.

Reply:
fixed